pkg/config/config.go

```diff
@@ -52,36 +53,16 @@ func (c *Config) GetHomeDir() (string, error) {
 	return porterDir, nil
 }
 
-// GetTemplatesDir determines the path to the templates directory.
-func (c *Config) GetTemplatesDir() (string, error) {
-	home, err := c.GetHomeDir()
-	if err != nil {
-		return "", err
-	}
-	return filepath.Join(home, "templates"), nil
-}
-
 // GetPorterConfigTemplate reads templates/porter.yaml from the porter home directory.
```

Comment:
Godoc/function comments here and elsewhere could use updates (no longer fetching from the porter home directory)